src/Microsoft.AspNetCore.Server.Kestrel.Https/HttpsConnectionFilter.cs

```diff
@@ -81,8 +91,21 @@ public HttpsConnectionFilter(HttpsConnectionFilterOptions options, IConnectionFi
 
                             return true;
                         });
-                    await sslStream.AuthenticateAsServerAsync(_options.ServerCertificate, clientCertificateRequired: true,
-                        enabledSslProtocols: _options.SslProtocols, checkCertificateRevocation: _options.CheckCertificateRevocation);
+
+                    certificateRequired = true;
+                }
+
+                context.Connection = sslStream;
+
+                try
+                {
+                    await sslStream.AuthenticateAsServerAsync(_options.ServerCertificate, certificateRequired,
+                            _options.SslProtocols, _options.CheckCertificateRevocation);
+                }
+                catch (IOException ex)
+                {
+                    _logger.LogInformation(1, ex, "Failed to authenticate HTTPS connection.");
```

Comment:
Where do we put the constant? A special HttpsTrace class? We don't make the IDs constants in KestrelTrace.